### PR TITLE
DROOLS-1781: replaces findbugs-maven-plugin

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Comment by mbiarnes** [comment]
Jenkins please retest this

File: business-central-parent/business-monitoring-webapp/pom.xml [hunk 1]
```diff
@@ -1390,6 +1390,15 @@
...
 
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
```
Comment (mareknovotny) [inline business-central-parent/business-monitoring-webapp/pom.xml]: Why we have this here, isn't it duplicating what is in https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/921 ?

File: business-central-parent/business-central-webapp/pom.xml [hunk 2]
```diff
@@ -2183,6 +2183,15 @@
...
 
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
```
Comment (mareknovotny) [inline business-central-parent/business-central-webapp/pom.xml]: Why we have this here, isn't it duplicating what is in [kiegroup/droolsjbpm-build-bootstrap#921](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/921) ?

File: business-central-parent/pom.xml [hunk 3]
```diff
@@ -51,10 +51,10 @@
     <pluginManagement>
       <plugins>
         <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>findbugs-maven-plugin</artifactId>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
           <configuration>
-            <excludeFilterFile>${project.basedir}/../../src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
+            <excludeFilterFile>${project.basedir}/../../src/main/spotbugs/spotbugs-exclude.xml</excludeFilterFile>
```
Comment (mareknovotny) [inline business-central-parent/pom.xml]: do we need this exclusion in distribution?
I removed that filtering and it works without any filter

**Comment by mbiarnes** [comment]
@mareknovotny latest changes - locally it works

**Comment by mbiarnes** [comment]
Jenkins please retest this

File: business-central-parent/pom.xml (hunk 3 above)
Comment (mareknovotny) [reply business-central-parent/pom.xml]: @mbiarnes  ^^^ have you tried to remove completely the spotbugs plugin here? Do we need it in distribution?

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Comment by mbiarnes** [comment]
@mareknovotny finally green with allchanges - so this PR and all related PRs can be merged

**Pull request merged**